### PR TITLE
Add gtm template

### PR DESCRIPTION
**Pull request by ilfa**

### CODE REVIEW

File: template.tpl
Comment (makma) [inline template.tpl:379]: Will this fail when we update the template version?
edit: I assume it won't since it uses mocked data later.

**Review by makma — APPROVED** [review]
LGTM for internal beta, once it's approved internally and tested we need to code review whole repo.

File: template.tpl
Comment (makma) [inline template.tpl]:
```suggestion
  "description": "This template adds FingerprintJS Pro agent, the highest accuracy device identification. Visit https://fingerprintjs.com/ for more information and get you public API key.",
```

File: metadata.yaml
Comment (makma) [inline metadata.yaml]:
```suggestion
documentation: "https://dev.fingerprintjs.com/docs/fingerprintjs-pro-google-tag-manager"
```

File: template.tpl
Comment (ilfa) [reply template.tpl:379]: It will, but I don't know better solution =(

**Pull request merged**
